src/app/shared/components/child-card/child-card.component.html

```diff
@@ -1,15 +1,35 @@
-<div>
+<div *ngIf="child">
+  <div fxLayout="row" fxLayoutAlign="end start">
+    <div class="actions_btn"><img src="assets/icons/edit.png" alt="edit" (click)="onEdit()"></div>
+    <div class="actions_btn"><img src="assets/icons/delete.png" alt="delete" (click)="onDelete()"></div>
+  </div>
   <mat-card>
-    <div class="image">
-      <img mat-card-image  alt="child photo">
-    </div>
-    <div class="content">
-      <mat-card-title>{{child.lastName}} {{child.firstName}} {{child.middleName}}</mat-card-title>
-    <mat-card-content>
-      <p>
-        {{child.dateOfBirth}}
-      </p>
-    </mat-card-content>
+    <div class="card" fxLayout="column" fxLayoutAlign="space-evenly space-between">
+      <div fxLayout="row" fxLayoutAlign="center">
+        <img class="card-image" mat-card-image src="assets/icons/child-{{child.gender}}.png" alt="child-photo">
+      </div>
+      <div>
+        <mat-card-header>
+          <mat-card-title class="card-title">{{child.lastName}} {{child.firstName}} {{child.middleName}}
+          </mat-card-title>
+        </mat-card-header>
+        <mat-card-content fxLayout="column" fxLayoutAlign="space-between start">
+          <p class="card_text"><img src="assets/icons/calendar.png" alt="calendar"> {{ child.dateOfBirth }}
```

Comment:
try to make a decision already to use indentations between brackets or not to use